tools/commands/test.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import argparse
+import os
+import subprocess
+
+logger = logging.getLogger("test")
+test_targets=['start', 'stop', 'restart', 'status', 'check']
+
+class prpltester(object):
```

Comment:
I don't think it's useful to make this a separate class, just merge it into the maptest class.

Reply:
okay, I was thinking that there will be more tests in the future but I will remove a class for now.